hs/server.hs

```diff
@@ -69,17 +79,38 @@ main = do
              dir graph $ graphResponse,
              dir code $ seeOther fbAuth1Url $ toResponse test,
              dir test $ look "code" >>= getEmail,
+             dir testPost $ look "code" >>= postToFacebook,
              dir about $ aboutResponse,
              dir static $ serveDirectory EnableBrowsing [] staticDir,
-             dir course $ path (\s -> liftIO $ queryCourse s)
+             dir course $ path (\s -> liftIO $ queryCourse s),
+             dir postFB $ seeOther fbAuth1UrlPost $ toResponse test
            ]
 
+postToFacebook :: String -> ServerPart Response
+postToFacebook code = (liftIO $ performPost code) >> graphResponse
+
+args2 :: FB.Argument
+args2 = ("message", "Test post please ignore")
+
+performPost :: String -> IO Response
+performPost code = withManager $ \manager -> FB.runFacebookT app manager $ do
```

Comment:
I did mean the "withManager" part too, although your change is also appreciated.

It shouldn't be too hard to define a function that basically does `withManager $ \manager -> FB.runFacebookT app manager` and takes in the rest as an argument.